examples/hpc/dla-driver/src/mmap.rs

```diff
@@ -116,9 +117,9 @@ pub(crate) const DLA_PP_INPUT_HEIGHT_BITMASK: usize = 0b111111111000000000000000
 
 pub(crate) const DLA_BUF_DATA_BANK: usize = 0x2C;
 pub(crate) const DLA_BUF_DATA_BANK_A_OFFSET: usize = 0x0;
-pub(crate) const DLA_BUF_DATA_BANK_B_OFFSET: usize = 0x10;
+pub(crate) const DLA_BUF_DATA_BANK_B_OFFSET: usize = 16;
 pub(crate) const DLA_BUF_DATA_BANK_A_BITMASK: usize = 0b1111;
-pub(crate) const DLA_BUF_DATA_BANK_B_BITMASK: usize = 0b11110000;
+pub(crate) const DLA_BUF_DATA_BANK_B_BITMASK: usize = 0b11110000000000000000;
```

Comment:
I think `0b1111 << 16` is going to be more readable here.

Reply:
Yeah this is way better. Modified longer bitmasks to use this syntax.